src/api.raml

```diff
@@ -837,6 +847,42 @@ traits:
                                 complete:
                                     displayName: Complete
                                     value: !include samples/digest/v1/complete.json
+        put:
+            description: |
+                Upsert a digest.
```

Comment:
Where in the dictionary will I find `"upsert"`? :wink:

Reply:
`(verb, adjective, noun): to cause discomfort to a data storage entity e.g. the database table was upsert that the INSERT query hurt her feelings`